dataservice/api/task/models.py

```diff
@@ -0,0 +1,67 @@
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.associationproxy import association_proxy
+
+from dataservice.extensions import db
+from dataservice.api.common.model import Base, KfId
+
+
+class Task(db.Model, Base):
+    """
+    Task entity represents an executed informatics task
+
+    :param kf_id: Unique id given by the Kid's First DCC
+    :param created_at: Time of object creation
+    :param modified_at: Last time of object modification
+    :param name: Name of the task
+    :param external_task_id: Id of executed task assigned by the external
+        informatics tool, eg: Cavatica or Firecloud.
+    """
+    __tablename__ = 'task'
+    __prefix__ = 'TB'
```

Comment:
Just curious why prefix isn't TA or TK? What's the B mean?

Reply:
TA is used by the release coordinator. We can go with TK